### PR TITLE
chore: update unit test and databenderror etc.

### PR DESCRIPTION
**Pull request by hanyisong**

### CODE REVIEW

**Comment by hanyisong** [comment]
@mrigger My local mvn verify is successfully

File: README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ Since SQL dialects differ widely, each DBMS to be tested requires a separate imp
 | TDEngine                     | Removed     | Untyped                      | We removed the TDEngine implementation since all but one of our bug reports were still unaddressed five months after we reported them.                                                          |
 | OceanBase                    | Working     | Untyped                      |                                                                                                                                                                                                 |
 | YugabyteDB                   | Working     | Typed (YSQL), Untyped (YCQL) | YSQL implementation based on Postgres code. YCQL implementation is primitive for now and uses Cassandra JDBC driver as a proxy interface.                                                       |
-
+| Databend                    | Working     | Typed                      |
```
Comment (mrigger) [inline README.md:64]: It seems that this line lacks a column (i.e., a `|`). Could you please also use the same indents as the other implementations?

File: src/sqlancer/databend/DatabendErrors.java [hunk 2]
```diff
@@ -8,109 +8,25 @@ private DatabendErrors() {
...
-
-        errors.add("does not have a column named"); // TODO: this only happens for views whose underlying table has a
-                                                    // removed column
-        errors.add("Contents of view were altered: types don't match!");
-        errors.add("Not implemented: ROUND(DECIMAL, INTEGER) with non-constant precision is not supported");
-        // TODO Databend待修复的bug（union schema error mismatch）
-        errors.add("unexpected end of file (failed to fill whole buffer)");
-    }
-
-    private static void addRegexErrors(ExpectedErrors errors) {
```
Comment (mrigger) [inline src/sqlancer/databend/DatabendErrors.java:58]: I'm a bit surprised. Why are all these unexpected errors no longer necessary?

**Review by mrigger — COMMENTED** [review]
Thanks a lot for the follow-up PR! Could you please take a look at the two comments and respond to them/address them? Could you please also rebase the commits to avoid the two initial merge commits?

**Comment by hanyisong** [comment]
@mrigger Why did I ignore .settings/org.eclipse.jdt.core.prefs and it still appears in my git cache. 
image

**Comment by mrigger** [comment]
I think to prevent this, we need to delete the `.settings` order and commit the change. If you'd like, you could create a follow-up PR for this.

Thanks for this PR, merging it now.

**Comment by hanyisong** [comment]
@mrigger I have tried deleting .settings on this pr, but can't pass the ci test.

**Pull request merged**